lib/galaxy/queue_worker.py

```diff
@@ -142,14 +141,7 @@ def recalculate_user_disk_usage(app, **kwargs):
     sa_session = app.model.context
     if user_id:
         user = sa_session.query( app.model.User ).get( app.security.decode_id( user_id ) )
-        if user:
-            if sa_session.get_bind().dialect.name not in ( 'postgres', 'postgresql' ):
-                new = user.calculate_disk_usage()
-            else:
-                new = pgcalc(sa_session, user.id)
-            user.set_disk_usage(new)
-            sa_session.add(user)
-            sa_session.flush()
```

Comment:
I removed it because I'm not sure how it'd possibly happen, but I guess it can't hurt.